### ISSUE TITLE
Custom Optimizer Example

### ISSUE THREAD

**Issue by Rishit-dagli** [Rishit-dagli/Gradient-Centralization-TensorFlow#16]
It would be great to include an example showing the use of this package with a custom optimizer built on top of `tf.keras.Optimizer` class.

**Comment by Rishit-dagli** [comment]
On another note I made an example demonstrating this but I soon realized it did not hold anything to offer which the existing two examples do not already (though the existing examples are quite similar themselves I thought it would be better to have at the minimum 2 examples). The code to use `gctf` is almost the same with a custom optimizer, further, the README also mentions the process for custom optimizers.
